sp800-63a/sec7_security.md

```diff
@@ -21,9 +21,13 @@ The threats to the enrollment process include impersonation attacks and threats
 |Enrollment | Falsified identity proofing evidence | An applicant claims an incorrect identity by using a forged driver's license.|
 | | Fraudulent use of another's identity | An applicant uses a passport associated with a different individual
 | | Repudiation of enrollment | A subscriber denies enrollment, claiming that he or she did not enroll with the CSP.|
+||Social engineering|A person manipulates an individual responsible for identity proofing to convince them to be enrolled as another individual.
```

Comment:
Ambiguous who 'them' applies to. Suggest using 'applicant' in place of 'person' and 'them'.